src/test/java/org/sagebionetworks/web/unitclient/presenter/QuizPresenterTest.java

```diff
@@ -63,15 +68,18 @@ public class QuizPresenterTest {
 	@Before
 	public void setup() throws JSONObjectAdapterException {
 		mockView = mock(QuizView.class);
+		mockSRCView = mock(CertificationQuizView.class);
 		mockSynapseClient = mock(SynapseClientAsync.class);
 		mockAuthenticationController = mock(AuthenticationController.class);
 		mockGlobalApplicationState = mock(GlobalApplicationState.class);
 		mockQuestionContainer = mock(QuestionContainerWidget.class);
 		mockInjector = mock(PortalGinInjector.class);
 		mockPassingRecord = mock(PassingRecord.class);
 		mockQuestionResponse = mock(MultichoiceResponse.class);
+		mockCookieProvider = mock(CookieProvider.class);
+		when(mockInjector.getCookieProvider()).thenReturn(mockCookieProvider);

```

Comment:
Expect for all existing tests, so I'd expect this to be in the before: 
 when(mockCookieProvider.getCookie(DisplayUtils.SYNAPSE_TEST_WEBSITE_COOKIE_KEY)).thenReturn(null)

There should be a new test though, where this cookie is set and you verify your new SRC view is used.